docs/userguide/campaigns.md

```diff
@@ -171,8 +156,8 @@ form:
 For discrete parameters, the parameter values associated with the provided measurements
 are required to fall into a predefined tolerance interval by default, which is
 defined on the level of the individual parameters.
-This requirement can be disabled using the 
-`numerical_measurements_must_be_within_tolerance` flag of the campaign.
+This requirement can be disabled using the method's
+`numerical_measurements_must_be_within_tolerance` flag.
```

Comment:
just to remember: once the tolerance stuff is completed we should have a separate section in this file (or in parameters?) explaining how we define tolerance and what you're actually controlling with the flag (as per our discussion)

Reply:
Absolutely. All changes here just stick with the current logic of the PR, in line with the bite-sized request. Following that logic, we'll natural encounter these "ghost changes" where we've already decided to go two steps further. But I think that's OK. If so, please resolve.